### ISSUE TITLE
Heroku Build - Error: Module not found: Error: Can't resolve 'react-tooltip'

### ISSUE THREAD

**Issue by gustavorps** [strapi/strapi#9382]
### Describe the bug

Deployment on Heroku fail with this error: Error: Module not found: Error: Can't resolve 'react-tooltip' in 'my-project/.cache/plugins/strapi-plugin-content-manager/admin/src/components/Tooltip'

<details>

### Log
<details><summary>See Heroku stdout</summary>

```
Enumerating objects: 36, done.
Counting objects: 100% (36/36), done.
Delta compression using up to 4 threads
Compressing objects: 100% (27/27), done.
Writing objects: 100% (29/29), 153.93 KiB | 1.59 MiB/s, done.
Total 29 (delta 17), reused 0 (delta 0)
remote: Compressing source files... done.
remote: Building source:
remote: 
remote: -----> Building on the Heroku-20 stack
remote: -----> Node.js app detected
remote:        
remote: -----> Creating runtime environment
remote:        
remote:        NPM_CONFIG_LOGLEVEL=error
remote:        NODE_VERBOSE=false
remote:        NODE_ENV=development
remote:        NODE_MODULES_CACHE=true
remote:        
remote: -----> Installing binaries
remote:        engines.node (package.json):  =12.19.0
remote:        engines.npm (package.json):   >=6.0.0
remote:        
remote:        Resolving node version =12.19.0...
remote:        Downloading and installing node 12.19.0...
remote:        Bootstrapping npm >=6.0.0 (replacing 6.14.8)...
remote:        npm >=6.0.0 installed
remote:        
remote: -----> Restoring cache
remote:        Cached directories were not restored due to a change in version of node, npm, yarn or stack
remote:        Module installation may take longer for this build
remote:        
remote: -----> Installing dependencies
remote:        Installing node modules
remote:        
remote:        added 1773 packages, and audited 1774 packages in 2m
remote:        
remote:        found 0 vulnerabilities
remote:        
remote: -----> Build
remote:        Running build
remote:        
remote:        > doisge-strapi@0.1.0 build
remote:        > strapi build
remote:        
remote:        Building your admin UI with development configuration ...
remote:        ℹ Compiling Webpack
remote: Error: Module not found: Error: Can't resolve 'react-tooltip' in '/tmp/build_be9566c4/.cache/plugins/strapi-plugin-content-manager/admin/src/components/Tooltip'
remote:     at /tmp/build_be9566c4/node_modules/strapi-admin/index.js:71:23
remote:     at finalCallback (/tmp/build_be9566c4/node_modules/webpack/lib/Compiler.js:257:39)
remote:     at onCompiled (/tmp/build_be9566c4/node_modules/webpack/lib/Compiler.js:265:20)
remote:     at /tmp/build_be9566c4/node_modules/webpack/lib/Compiler.js:670:21
remote:     at eval (eval at create (/tmp/build_be9566c4/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:26:1)
remote:     at /tmp/build_be9566c4/node_modules/webpack/lib/Compilation.js:1173:13
remote:     at /tmp/build_be9566c4/node_modules/webpack/lib/Compilation.js:1096:25
remote:     at /tmp/build_be9566c4/node_modules/webpack/lib/Compilation.js:1017:13
remote:     at /tmp/build_be9566c4/node_modules/neo-async/async.js:2830:7
remote:     at /tmp/build_be9566c4/node_modules/neo-async/async.js:2830:7
remote:     at done (/tmp/build_be9566c4/node_modules/neo-async/async.js:2863:11)
remote:     at /tmp/build_be9566c4/node_modules/neo-async/async.js:2818:7
remote:     at /tmp/build_be9566c4/node_modules/webpack/lib/Compilation.js:1017:13
remote:     at /tmp/build_be9566c4/node_modules/neo-async/async.js:2830:7
remote:     at /tmp/build_be9566c4/node_modules/neo-async/async.js:2830:7
remote:     at done (/tmp/build_be9566c4/node_modules/neo-async/async.js:2863:11)
remote: npm ERR! code 1
remote: npm ERR! path /tmp/build_be9566c4
remote: npm ERR! command failed
remote: npm ERR! command sh -c strapi build
remote: 
remote: npm ERR! A complete log of this run can be found in:
remote: npm ERR!     /tmp/npmcache.O6XZE/_logs/2021-02-11T14_24_50_555Z-debug.log
remote: 
remote: -----> Build failed
remote:        
remote:        We're sorry this build is failing! You can troubleshoot common issues here:
remote:        https://devcenter.heroku.com/articles/troubleshooting-node-deploys
remote:        
remote:        If you're stuck, please submit a ticket so we can help:
remote:        https://help.heroku.com/
remote:        
remote:        Love,
remote:        Heroku
remote:        
remote:  !     Push rejected, failed to compile Node.js app.
remote: 
remote:  !     Push failed
remote:  !
remote:  ! ## Warning - The same version of this code has already been built: 1b0d43364c9d57134508e985870093cfb66540a6
remote:  !
remote:  ! We have detected that you have triggered a build from source code with version 1b0d43364c9d57134508e985870093cfb66540a6
remote:  ! at least twice. One common cause of this behavior is attempting to deploy code from a different branch.
remote:  !
remote:  ! If you are developing on a branch and deploying via git you must run:
remote:  !
remote:  !     git push heroku <branchname>:main
remote:  !
remote:  ! This article goes into details on the behavior:
remote:  !   https://devcenter.heroku.com/articles/duplicate-build-version
remote: 
remote: Verifying deploy...
remote: 
remote: !       Push rejected
```
</details>

### Steps to reproduce the behavior

1. `yarn create strapi-app my-project --quickstart`
1. Follow [Heroku deployment guide from Strapi](https://strapi.io/documentation/developer-docs/latest/deployment/heroku.html)
1. Wait Heroku build after `git push heroku main`
1. See error

### Expected behavior
Generate build

### System

- Node.js version: v12.19.0
- NPM version: 6.14.8
- Strapi version: 3.4.5
- Database: SQLite
- Operating system: Linux 5.4.0-3-amd64 #1 SMP Debian 5.4.13-1~mx19+1 (2020-01-19) x86_64 GNU/Linux

**Comment by derrickmehaffy** [comment]
Duplicate: https://github.com/strapi/strapi/issues/9442

Fix will be released in v3.5.1 this week

**Comment by mrmikardo** [comment]
I'm still having this problem with Strapi version `3.6.0`. Any idea how I can resolve it?
